string cluster_label
geometry_msgs/Pose pose
int64 number_people
